DownFall/LevelCoordinator.swift

```diff
@@ -80,7 +83,7 @@ class LevelCoordinator: LevelCoordinating {
         let nextArea = runModel.currentArea()
         switch nextArea.type {
         case .level(let level):
-            presentNextLevel(level, playerData: runModel.player)
+            presentNextLevel(level, playerData: runModel.player, savedTiles: runModel.savedTiles)
```

Comment:
presentNextLevel could just take a runModel

Reply:
probably easier to leave it as is.  it makes it easier to test this especially if RunModel gets larger